### PR TITLE
documentation updates

### PR DESCRIPTION
**Pull request by julieh11**

### CODE REVIEW

File: configs/1d/defaults.yaml
Comment (almilder) [inline configs/1d/defaults.yaml:1]: Please leave this unchanged

File: configs/1d/inputs.yaml
Comment (almilder) [inline configs/1d/inputs.yaml:1]: Please leave this unchanged

File: docs/source/_elfolder/electron_fit_ranges.JPG
Comment (almilder) [inline docs/source/_elfolder/electron_fit_ranges.JPG:1]: Is this used? it looks like there is an annotated version and a non-annotated version

File: docs/source/_elfolder/electron_fit_annotated.PNG
Comment (almilder) [inline docs/source/_elfolder/electron_fit_annotated.PNG:1]: Can you fix the colorbar axis here?

File: docs/source/_elfolder/epw_best.png
Comment (almilder) [inline docs/source/_elfolder/epw_best.png:1]: Delete

Comment (julieh11) [reply docs/source/_elfolder/epw_best.png:1]: This is used in the tutorial, are you sure you wan to delete?

File: docs/source/_elfolder/iaw_worst.png
[the remark this answers is not in the document]
Comment (julieh11) [reply docs/source/_elfolder/iaw_worst.png:1]: Used in the tutorial

**Pull request closed without merging**